### PR TITLE
chore(): use current node lts release

### PR DESCRIPTION
**Pull request by gmq**

### CODE REVIEW

**Review by difernandez — COMMENTED** [review]
I think there shouldn't be any problem merging this, @rjherrera could you confirm though? Wanna make sure there's nothing I'm missing, considering #366. Regardless, a couple of comments:

- We should change the `.node-version`
- Maybe we should do for potassium what it already does in the `config.yml` for generated projects: download the node from nodesource using potassiums .nod-version, instead of relaying in the ruby node image. We could do this in another PR, wanted to know what you guys think

**Comment by rjherrera** [comment]
> I think there shouldn't be any problem merging this, @rjherrera could you confirm though? Wanna make sure there's nothing I'm missing, considering #366. Regardless, a couple of comments:
> 
> * We should change the `.node-version`
> * Maybe we should do for potassium what it already does in the `config.yml` for generated projects: download the node from nodesource using potassiums .nod-version, instead of relaying in the ruby node image. We could do this in another PR, wanted to know what you guys think

You mean potassium's `.node-version` right? Cause the one generated for each project [is done automatically using the version from `version.rb`](https://github.com/platanus/potassium/blob/master/lib/potassium/recipes/node.rb#L8). I agree with the second point, but yeah, for another one.

**Review by rjherrera — APPROVED** [review]

**Comment by difernandez** [comment]
> You mean potassium's .node-version right? 
Yes! It's currently in 12, and I think `circleci/ruby:2.7.3-node` uses node 14 also

**Review by difernandez — APPROVED** [review]

**Comment by rjherrera** [comment]
> Yes! It's currently in 12, and I think `circleci/ruby:2.7.3-node` uses node 14 also

Right, yeah I would upgrade that too. And now that I think of it, it should be done in this PR, because the [`node version ensurer`](https://github.com/platanus/potassium/blob/master/lib/potassium/node_version_ensurer.rb) uses the version from the `version.rb` file to check, so it should be consistent

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-ruby-image: &ruby-image circleci/ruby:2.7.0-node
+ruby-image: &ruby-image circleci/ruby:2.7.3-node
 env-vars: &env-vars
   BUNDLE_JOBS: 4
   RAILS_ENV: test
   RAKE_ENV: test
   BUNDLE_PATH: /home/circleci/project/vendor/bundle
-  GEM_PATH: /home/circleci/project/vendor/bundle/ruby/2.7.0
+  GEM_PATH: /home/circleci/project/vendor/bundle/ruby/2.7.3
```
Comment (rjherrera) [inline .circleci/config.yml:9]: esto funciona bien? juraría que las rutas de gemas se mantienen con el .0 a pesar de que la versión que uno use sea la .x 🤔  @difernandez te acordai?

Comment (difernandez) [reply .circleci/config.yml:9]: damn, no, no me acuerdo 🤔 . Vi en local, y no se si es extrapolable, pero ahí para la versión `2.6.2` las gemas están en el path `~/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0`, entonces me hace sentido que se mantenga el `.0` en el patch

Comment (rjherrera) [reply .circleci/config.yml:9]: me queda la duda (y me ha costado encontrar en google) si ese 2.6.0 tuyo (y que tmb lo vi en mi pc) se corresponde con el de la linea, porque hay uno que es con patch y uno que es sin patch jaja

Comment (difernandez) [reply .circleci/config.yml:9]: corrí de nuevo el job con ssh y efectivamente, es 2.7.0:
image

**Review by difernandez — APPROVED** [review]

**Pull request merged**